app/scripts/controllers/preferences-controller.ts

```diff
@@ -17,11 +25,90 @@ const testNetworks = {
   [CHAIN_IDS.LINEA_SEPOLIA]: true,
 };
 
+type PreferencesControllerOptions = {
+  networkConfigurations?: Record<string, { chainId: Hex }>;
+  initState?: Partial<PreferencesControllerState>;
+  initLangCode?: string;
+  // TODO: Replace `any` with type
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  network?: any;
```

Comment:
I believe the `network` property can just be removed I don't see it used anywhere.